packages/builder/src/builderStore/store/frontend.js

```diff
@@ -408,6 +409,9 @@ export const getFrontendStore = () => {
             return state
           }
 
+          // immediately need to remove bindings, currently these aren't valid when pasted
+          state.componentToPaste = removeBindings(state.componentToPaste)
```

Comment:
I think we only want to do this when not cutting, which is checked down below. The reason for this is in the issue comments, but basically we still want to be able to drag and drop to move components without invalidating the bindings.